app/src/main/java/com/yacgroup/yacguide/database/tourbook/TourbookEntryVerbose.kt

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2021 Christian Sommer
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package com.yacgroup.yacguide.database.tourbook
+
+import com.yacgroup.yacguide.database.Ascend
+import com.yacgroup.yacguide.database.DatabaseWrapper
+import com.yacgroup.yacguide.utils.AscendStyle
+import com.yacgroup.yacguide.utils.ParserUtils
+
+import kotlin.reflect.full.memberProperties
+
+class TourbookEntryVerbose(_ascend: Ascend, val _db: DatabaseWrapper) {
+    companion object {
+        /*
+         * Return a list of all property names of this class
+         * which are treated as fields to be exported.
+         */
+        fun keys(): List<String> {
+            return buildList {
```

Comment:
Since you don't have private members, this could be shortened to
`return TourbookEntryVerbose::class.memberProperties.map{ it.name }`

Reply:
Can `keys()` actually be private?

Reply:
> Can `keys()` actually be private?

No, because it used in the `TourbookExporter` class.